src/styles/variables.scss

```diff
@@ -0,0 +1,9 @@
+$cell-size: 100px;
+$border-width: 1px;
+$cell-font-size: 30px;
+$cell-padding: 10px;
+$num-font-family: arial, sans-serif;
+$cell-background: #eee;
+$cell-hover-color: pink;
```

Comment:
for colors value it is a good practice to name them as `$color-pink` or `$color-white` etc.., they should point only color not need to add hover or selector names, so please fix it for both variables